darwin/dataset/remote_dataset.py

```diff
@@ -251,16 +255,46 @@ def pull(
             If the given ``release`` has an invalid format.
         ValueError
             If darwin in unable to get ``Team`` configuration.
+        ValueError
+            If the release is still processing after the maximum retry duration.
         """
 
         console = self.console or Console()
 
         if release is None:
-            release = self.get_release()
+            if retry:
+                raise ValueError(
+                    "To retry downloading a release, a release name must be provided. This can be done as follows:\n\nrelease = dataset.get_release(name='release_name')\ndataset.pull(release=release, retry=True)"
+                )
+            else:
+                release = self.get_release(retry=retry)
 
         if release.format != "json" and release.format != "darwin_json_2":
             raise UnsupportedExportFormat(release.format)
 
+        if release.status == "pending":
+            if retry:
+                retry_duration = 300
+                retry_interval = 10
```

Comment:
It would be more conventional to have these configurable via CLI or some SDK settings

Reply:
@balysv This makes sense. I can see 2 options, both of which involve building in some validation:
- 1: Make these values configurable in the `~/.config.yaml` file, or:
- 2: Add two additional arguments: `retry_duration` and `retry_interval` with default values of ~10 minutes & ~10 seconds. These can be configured, but if they're passed without `retry=True` then we will throw an error

I'm leaning toward the additional arguments